lib/edge_commander/sites/records.ex

```diff
@@ -0,0 +1,25 @@
+defmodule EdgeCommander.Sites.Records do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias EdgeCommander.Sites.Records
+
+
+  schema "sites" do
+    field :location, :string
+    field :name, :string
+    field :notes, :string
+    field :nvr_id, :integer
+    field :router_id, :integer
+    field :sim_number, :integer
```

Comment:
change it from number to varchar or string https://stackoverflow.com/a/24353813/5650044